buildah/buildah.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: buildah
+spec:
+  parameters:
+  - name: BUILDER_IMAGE
+    description: The location of the buildah builder image.
+  - name: IMAGE
+    description: The name of the image to push.
+  - name: DOCKERFILE
+    description: Path to the Dockerfile to build.
+    default: ./Dockerfile
+
+  steps:
+  - name: build
+    image: ${BUILDER_IMAGE}
+    args: ['bud', '-f', '${DOCKERFILE}', '-t', 'image', '.']
```

Comment:
let's update this to `['--storage-driver=vfs', 'bud', '-f', '${DOCKERFILE}', '-t', 'image', '.']`